src/components/sectionList/listActions/SectionListActions.tsx

```diff
@@ -99,6 +103,18 @@ export const ActionMore = ({
                                 }}
                             ></MenuItem>
                         </TooltipWrapper>
+
+                        <MenuItem
```

Comment:
Also need to check here if user can has `delete` access. Should not be rendered (or should disable I'm not sure). Old app removes it from the list, but we might want to talk to Joe about it.

So it might be worth passing down an `Partial<Access>` object, or if we want this on the root, I'm not sure.

Reply:
Done

> Feel free to rename these

Renamed to `editable` and `deletable` (I kept the function name `canEditModel` though)